src/main/typescript/lib/metadata/Metadata.ts

```diff
@@ -178,7 +178,7 @@ export const metadataConverter: (input: any, dropDowns: DropdownLists) => Deposi
         // privacy sensitive data
         privacySensitiveDataPresent: privacySensitiveDataPresent,
 
-        // deposit license
+        // depositor agreement
         acceptLicenseAgreement: input.acceptLicenseAgreement || false,
```

Comment:
are you sure you don't want to change the fieldname in the data model as well? That would require changes in several files as well...

Reply:
I'll create a separate issue for it, as this requires changes in both API and UI.

Reply:
no, it doesn't. There is a mapping from external model to internal model, so changing the internal model's fields doesn't require changes to the external model and/or API